x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/Analyzer.java

```diff
@@ -700,6 +737,38 @@ private LogicalPlan resolveFork(Fork fork, AnalyzerContext context) {
             return new Fork(fork.source(), fork.child(), newSubPlans);
         }
 
+        private LogicalPlan resolveRerank(Rerank rerank, List<Attribute> childrenOutput) {
```

Comment:
ℹ️ Ensure resolution of the reranker fields and that the score field is present.